doc/examples/filters/plot_blind_deconvolution.py

```diff
@@ -0,0 +1,120 @@
+"""
+=========================
+Image blind Deconvolution
+=========================
+We present here a usage example of a blind deconvolution algorithms
+based on the Richardson-Lucy deconvolution algorithm ([1]_, [2]_).
+
+Blind deconvolution is the term given to an image-restoration technique in
+which complete knowledge of both the point-spread function (PSF) and the
+object are not available.
+
+.. [1] Fish, D. A., A. M. Brinicombe, E. R. Pike, and J. G. Walker.
+       "Blind deconvolution by means of the Richardson–Lucy algorithm."
+       JOSA A 12, no. 1 (1995): 58-65.
+
+.. [2] https://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
+"""
+
+
+import numpy as np
+from scipy.signal import convolve2d
+import matplotlib.pyplot as plt
+from skimage.filters import gaussian
+from skimage.restoration import richardson_lucy
+
+
+# Initialize image that is to be recovered (cross)
+noisy_image = np.zeros((100, 100), dtype=np.float32)
+noisy_image[40:60, 45:55] = 1
+noisy_image[45:55, 40:60] = 1
+
+# Add some poisson photon shot noise
+np.random.seed(0)
+noisy_image += np.random.poisson(2.0, noisy_image.shape) / 255
+
+# Create the PSF
+psf_gaussian = np.zeros_like(noisy_image)
+w, h = noisy_image.shape
```

Comment:
the names are misleading here, if they mean width and height, since it's the other way around.

Reply:
I think you can just invert them :-)